input-method.php

```diff
@@ -5,6 +5,7 @@
     echo "<?xml version=\"1.0\" encoding=\"UTF-8\"?>";
     
     $searchType = $_REQUEST['Digits'];
+    $inputTitle = $_REQUEST['Title'];
```

Comment:
There is already a "Title" configuration value.  We should use that, but instead specify that we want to play the title here.  Also you need to check to make sure this value is set to `1`, otherwise it will bomb out.

```php
$playTitle = isset($_REQUEST['PlayTitle']) ? $_REQUEST['PlayTitle'] : 0;
```

Reply:
Be sure to also add the documentation of this feature in README.md.